### ISSUE TITLE
Support to use a custom date picker when adding a new media entry.

### ISSUE THREAD

**Issue by AB1908** [mProjectsCode/obsidian-media-db-plugin#81]
I usually add media entries via templater. Fairly tired of doing it and would like to switch over.

Among other requirements, I sometimes add an entry a few days after I've watched it. I don't want to edit this by hand and instead would like to get a date picker popup to select the correct timestamp/date. 

Is this possible? I'd rather not use an additional dep like Metabind but curious to hear your thoughts.

**Comment by mProjectsCode** [comment]
Normally i would say to you: "Just use meta bind" xD

But the plugin doesn't add a `watchedOn` property anyways.

**Comment by AB1908** [comment]
Fair enough. I'll work on some changes in a fork in the upcoming months. I'm tired of fixing my script lol.

**Comment by mProjectsCode** [comment]
So you are tired of fixing your script, so you switch to fixing my script? xD
